app/build.gradle

```diff
@@ -58,18 +58,15 @@ dependencies {
     testImplementation 'org.robolectric:robolectric:3.7.1'
     testImplementation 'com.nhaarman:mockito-kotlin:1.5.0'
     testImplementation 'com.squareup.okhttp3:mockwebserver:3.8.1'
-
```

Comment:
Please do not improve code quality if it is irrelevant with the issue being solved. I think these spaces were intentionally there to increase readability. Can you please revert your commit? Or add spaces again?